mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoInsertMethodGenerator.java

```diff
@@ -127,8 +129,21 @@ public DaoInsertMethodGenerator(
 
     List<? extends VariableElement> parameters = methodElement.getParameters();
     String entityParameterName = parameters.get(0).getSimpleName().toString();
-    NullSavingStrategy nullSavingStrategy =
-        methodElement.getAnnotation(Insert.class).nullSavingStrategy();
+
+    Insert annotation = methodElement.getAnnotation(Insert.class);
```

Comment:
This code is repeated among the next four classes, could probably be refactored similarly to how I describe in `NullSavingStrategyValidation`

Reply:
https://github.com/datastax/java-driver/pull/1246/commits/6d1e5e67a568b52db19fbc6a587288a9a593894e